js-src/Settings.ts

```diff
@@ -0,0 +1,210 @@
+ */
+export function getSettingsJson(): string {
+  return neon.getSettingsJson();
+}
+
+/**
+ * Load trust configuration into the main trust settings.
+ * @param trustConfig The trust configuration object
+ */
+export function loadTrustConfig(trustConfig: TrustConfig): void {
+  // Convert camelCase to snake_case for Rust compatibility
+  // Explicitly set undefined fields to null to ensure they are cleared
+  const rustConfig = {
+    verify_trust_list: trustConfig.verifyTrustList,
+    user_anchors: trustConfig.userAnchors ?? null,
+    trust_anchors: trustConfig.trustAnchors ?? null,
+    trust_config: trustConfig.trustConfig ?? null,
+    allowed_list: trustConfig.allowedList ?? null,
+  };
+  neon.loadTrustConfig(JSON.stringify(rustConfig));
+}
+
+/**
+ * Load trust configuration into the CAWG trust settings.
+ * @param trustConfig The trust configuration object
+ */
+export function loadCawgTrustConfig(trustConfig: TrustConfig): void {
+  // Convert camelCase to snake_case for Rust compatibility
+  // Explicitly set undefined fields to null to ensure they are cleared
+  const rustConfig = {
+    verify_trust_list: trustConfig.verifyTrustList,
+    user_anchors: trustConfig.userAnchors ?? null,
+    trust_anchors: trustConfig.trustAnchors ?? null,
+    trust_config: trustConfig.trustConfig ?? null,
+    allowed_list: trustConfig.allowedList ?? null,
+  };
+  neon.loadCawgTrustConfig(JSON.stringify(rustConfig));
+}
+
+/**
+ * Get the current trust configuration.
+ * @returns The current trust configuration object
+ */
+export function getTrustConfig(): TrustConfig {
+  const json = neon.getTrustConfig();
+  const parsed = JSON.parse(json);
+
+  // Convert snake_case to camelCase to match TrustConfig interface
+  // Handle undefined values by converting them to null
+  return {
+    verifyTrustList: parsed.verify_trust_list,
+    userAnchors: parsed.user_anchors ?? null,
+    trustAnchors: parsed.trust_anchors ?? null,
+    trustConfig: parsed.trust_config ?? null,
+    allowedList: parsed.allowed_list ?? null,
+  };
+}
+
+/**
+ * Get the current CAWG trust configuration.
+ * @returns The current CAWG trust configuration object
+ */
+export function getCawgTrustConfig(): TrustConfig {
+  const json = neon.getCawgTrustConfig();
+  const parsed = JSON.parse(json);
+
+  // Convert snake_case to camelCase to match TrustConfig interface
+  // Handle undefined values by converting them to null
+  return {
+    verifyTrustList: parsed.verify_trust_list,
+    userAnchors: parsed.user_anchors ?? null,
+    trustAnchors: parsed.trust_anchors ?? null,
+    trustConfig: parsed.trust_config ?? null,
+    allowedList: parsed.allowed_list ?? null,
+  };
+}
+
+/**
+ * Load verify configuration into the verify settings.
+ * @param verifyConfig The verify configuration object
+ */
+export function loadVerifyConfig(verifyConfig: VerifyConfig): void {
+  // Convert camelCase to snake_case for Rust compatibility
+  const rustConfig = {
```

Comment:
`rustConfig` because it's related to the Rust codebase, or type with the missing `t` for `trustconfig`?